unittests/test_lib_fork.py

```diff
@@ -109,11 +109,11 @@ def test_fetch_ready_fork_result_list(self):
         pipe_ids = {}
         expected = {}
         svl = 10
-        for key in range(1, 5):
-            pipe_ids[key] = fork_in_bg(sleep_fn, svl)
-            expected[key] = str(svl)
+        for i in range(1, 5):
+            pipe_ids[i] = fork_in_bg(sleep_fn, svl)
+            expected[i] = str(svl)
 
-        result = fetch_ready_fork_result_list(pipe_ids)
+        result = fetch_ready_fork_result_list({})
```

Comment:
Why the pipe_ids are removed here?

Reply:
pipe_ids = {}

Reply:
expected = {}

Reply:
if u just assigned it to pipe_ids will give you error
i have tried couple times and giving error
once i assigned it to {} it ran